CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.9.8
+
+* Remove dep on scheduled test. 
```

Comment:
I'd just write "Internal changes only" so that users don't have to spend mental energy thinking about whether this release is relevant to them.

Reply:
The changelog is an aspect of API documentation—it should be targeted at downstream users of the package. Details like this that are only relevant to package maintainers belong in the commit log.